deeprank/features/atomicfeature_1.py

```diff
@@ -0,0 +1,703 @@
+
+            param_vdw (str): file name of the force field containing
+                vdw parameters e.g. protein-allhdg5.4_new.param.
+                Must be of the format:
+                * NONBonded  CYAA    0.105   3.750       0.013    3.750
+                * NONBonded  CCIS    0.105   3.750       0.013    3.750
+
+            patch_file (str): file name of a valid patch file for
+                the parameters e.g. patch.top.
+                The way we handle the patching is very manual and
+                should be made more automatic.
+
+            contact_cutoff (float): the maximum distance in Å
+                between 2 contact atoms.
+
+            verbose (bool): print or not.
+
+        Examples:
+            >>> pdb = '1AK4_100w.pdb'
+            >>>
+            >>> # get the force field included in deeprank
+            >>> # if another FF has been used to compute the ref
+            >>> # change also this path to the correct one
+            >>> FF = pkg_resources.resource_filename(
+            >>>     'deeprank.features','') + '/forcefield/'
+            >>>
+            >>> # declare the feature calculator instance
+            >>> atfeat = AtomicFeature(pdb,
+            >>>    param_charge = FF + 'protein-allhdg5-4_new.top',
+            >>>    param_vdw    = FF + 'protein-allhdg5-4_new.param',
+            >>>    patch_file   = FF + 'patch.top')
+            >>>
+            >>> # assign parameters
+            >>> atfeat.assign_parameters()
+            >>>
+            >>> # only compute the pair interactions here
+            >>> atfeat.evaluate_pair_interaction(save_interactions=test_name)
+            >>>
+            >>> # close the db
+            >>> atfeat.sqldb._close()
+        """
+        super().__init__("Atomic")
+
+        # set a few things
+        self.pdbfile = pdbfile
+        self.mutant = mutant
+        self.param_charge = param_charge
+        self.param_vdw = param_vdw
+        self.patch_file = patch_file
+        self.contact_cutoff = contact_cutoff
+        self.verbose = verbose
+
+        # a few constant
+        self.eps0 = 1
+        self.c = 332.0636
+        self.residue_key = 'chainID, resSeq, resName'
+        self.atom_key = 'chainID, resSeq, resName, name'
+
+        # read the pdb as an sql
+        self.sqldb = pdb2sql.pdb2sql(self.pdbfile)
+
+        # read the force field
+        self.read_charge_file()
+
+        if patch_file is not None:
+            self.read_patch()
+        else:
+            self.patch_charge, self.patch_type = {}, {}
+
+        # read the vdw param file
+        self.read_vdw_file()
+        
+        # contact atoms to contact residues
+        self.extend_contact_to_residue()
+
+        
+
+    ####################################################################
+    #
+    #   READ INPUT FILES
+    #
+    ####################################################################
+
+    def read_charge_file(self):
```

Comment:
I thought we were going to make these file parsers more object-oriented. Didn't you say that you were going to use the parsers that I made earlier: https://github.com/DeepRank/deeprank/tree/residue-contacts/deeprank/parse
?